packages/react/src/DiagnosticReportDisplay/DiagnosticReportDisplay.tsx

```diff
@@ -70,60 +87,92 @@ export function DiagnosticReportDisplay(props: DiagnosticReportDisplayProps): JS
   return (
     <Stack>
       <Title>Diagnostic Report</Title>
-      <Group mt="md" spacing={30}>
-        {diagnosticReport.subject && (
-          <div>
+      {DiagnosticReportHeader(diagnosticReport)}
+      {!props.hideSpecimenInfo && SpecimenInfo(specimens)}
+      {diagnosticReport.result && (
+        <ObservationTable hideObservationNotes={props.hideObservationNotes} value={diagnosticReport.result} />
+      )}
+      {specimenNotes.length > 0 && <NoteDisplay value={specimenNotes} />}
+    </Stack>
+  );
+}
+
+function DiagnosticReportHeader(diagnosticReport: DiagnosticReport): JSX.Element {
```

Comment:
This should probably either be a normal function (`camelCase`) or a React component.  Is there a reason you broke it out rather than leaving it inline?